content/self-host/client-configuration/_index.en.md

```diff
@@ -45,7 +45,7 @@ The easiest way to setup automatically is using deployment scripts found [here](
 
 You can fix the Password is required and use a reverse Base64 string in the format `{"host":"HOSTADDRESS","key":"HOSTKEY","api":"http://HOSTADDRESS:21114"}` to automatically configure the clients, this is available automatically with RustDesk Server Pro via the console.
 
-You can also use the steps from [above](/docs/en/self-host/client-configuration/#setup-using-import-or-export) to export the string, remove any `=` at the start or end of the string.
+You can also use the steps from [above](/docs/en/self-host/client-configuration/#setup-using-import-or-export) to export the string, remove any `=` at the start or end of the string. Restart RustDesk services on the client if settings don't show.
```

Comment:
Should not be "restart RustDesk GUI"? No need to restart the service.

Reply:
I'll need to spin up a machine and check but did work here https://github.com/rustdesk/rustdesk/issues/5740

Reply:
Yeah from my tests its restarting the gui